mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+    options.
+
+    If present, streamMap is a dictionary of MPAS-O stream names that map to
+    their mpas_analysis counterparts.
+
+    If present, variableMap is a dictionary of MPAS-O variable names that map
+    to their mpas_analysis counterparts.
+
+    Authors: Mark Petersen, Phillip J. Wolfram, Milena Veneziani
+    Modified: 02/23/2017
+    """
+
+    # read parameters from config file
+    inDirectory = config.get('input', 'baseDirectory')
+
+    namelistFileName = config.get('input', 'oceanNamelistFileName')
+    namelist = NameList(namelistFileName, path=inDirectory)
+
+    streamsFileName = config.get('input', 'oceanStreamsFileName')
+    streams = StreamsFile(streamsFileName, streamsdir=inDirectory)
+
+    timeSeriesStatsMonthlyAnalysisMemberFlag = namelist.get(
+        'config_am_timeseriesstatsmonthly_enable')
+    if timeSeriesStatsMonthlyAnalysisMemberFlag == '.false.':
+        raise RuntimeError('*** MPAS-Analysis relies on the existence of '
+                           'monthly\n*** averaged files: make sure to '
+                           'enable the\n*** timeSeriesStatsMonthly '
+                           'analysis member!')
+
+    mocAnalysisMemberFlag = namelist.get(
+        'config_am_mocstreamfunction_enable')
+
+    calendar = namelist.get('config_calendar_type')
+    simulationStartTime = get_simulation_start_time(streams)
+
+    # get a list of timeSeriesStats output files from the streams file,
+    # reading only those that are between the start and end dates
+    streamName = streams.find_stream(streamMap['timeSeriesStats'])
+    #   First a list necessary for the streamfunctionMOC climatology
+    startDateClimo = config.get('climatology', 'startDate')
+    endDateClimo = config.get('climatology', 'endDate')
+    inputFilesClimo = streams.readpath(streamName,
+                                       startDate=startDateClimo,
+                                       endDate=endDateClimo,
+                                       calendar=calendar)
+    print '\n  List of files for climatologies:\n{} through {}'.format(
+           inputFilesClimo[0], inputFilesClimo[-1])
+    startYearClimo = config.getint('climatology', 'startYear')
+    endYearClimo = config.getint('climatology', 'endYear')
+
+    plotsDirectory = buildConfigFullPath(config, 'output', 'plotsSubdirectory')
+    mainRunName = config.get('runs', 'mainRunName')
+    #   Then a list necessary for the streamfunctionMOC Atlantic timeseries
+    startDateTseries = config.get('timeSeries', 'startDate')
+    endDateTseries = config.get('timeSeries', 'endDate')
+    inputFilesTseries = streams.readpath(streamName,
+                                         startDate=startDateTseries,
+                                         endDate=endDateTseries,
+                                         calendar=calendar)
+    print '  List of files for timeSeries:\n{} through {}'.format(
+           inputFilesTseries[0], inputFilesTseries[-1])
```

Comment:
It is probably better to leave as-is upon further reflection of our use cases.  We would use a function if we wanted to standardize output across all analysis, which may be worth doing.  Just something to think about...